.github/ISSUE_TEMPLATE/Bug_Report.yaml

```diff
@@ -21,7 +21,7 @@ body:
       required: true
     - label: Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
       required: true
-    - label: Refer to the [known issues](https://docs.microsoft.com/powershell/scripting/whats-new/known-issues-ps6).
+    - label: Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).
```

Comment:
This link seems to 404: https://docs.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell

Also a nitpick but I would capitalize the D in Differences since its in the middle of a sentent

Reply:
404 is expected. The URL will be live by 5pm PDT (if not sooner).

The `D` is capitalized because that is the title of the article. That is the style we use in docs.